src/CMakeLists.txt

```diff
@@ -14,15 +14,73 @@ target_include_directories(micm
     $<INSTALL_INTERFACE:include>
 )
 
+set(CMAKE_CUDA_ARCHITECTURES 80)
```

Comment:
Yes, please delete this. If we need a flag I can edit the cmake files to set this when we configure cmake

Reply:
@qinatan could you please delete this line and let us know if it affects the compilation?

Reply:
@K20shores Qina mentioned that she had a problem of moving this line to the `ENABLE_CUDA` section and it looks you just deleted it in the latest commit. Is this what we should move on with?

Reply:
Unclear. I was trying to test it yesterday but my request to get an interactive session on gust never completed. I'll attempt to test it out again this morning

Reply:
Aha, Gust is typically on maintenance every Tuesday. This morning should work. Let us know how your revision goes.

Reply:
@kyle. You should test it on **AddForcingTerm_CUDA branch** because that's the branch that uses atomicAdd() and requires set(CMAKE_CUDA_ARCHITECTURES 80). I will run it now and post an error message here if there's one.

Reply:
I moved set(CMAKE_CUDA_ARCHITECTURES 80) to the ENABLE_CUDA block at line 69. Here is the error message. It did not recognize the atomicAdd () function

/glade/derecho/scratch/qinatan/micm/src/process/process_set.cu(51): error: no instance of overloaded function "atomicAdd" matches the argument list
            argument types are: (double *, double)

/glade/derecho/scratch/qinatan/micm/src/process/process_set.cu(54): error: no instance of overloaded function "atomicAdd" matches the argument list
            argument types are: (double *, double)

2 errors detected in the compilation of "/glade/derecho/scratch/qinatan/micm/src/process/process_set.cu".
make[2]: *** [src/CMakeFiles/micm_cuda.dir/build.make:77: src/CMakeFiles/micm_cuda.dir/process/process_set.cu.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:518: src/CMakeFiles/micm_cuda.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
/var/spool/pbs/mom_priv/jobs/1242536.desched1.SC: line 19: ./test_cuda_process_set: No such file or directory

Reply:
I successfully built and ran this branch, flipped_memory, with it removed on gust just now. However, I don't see any atomicAdd function calls in the cuda code. Where is that code?

Reply:
atomicAdd function is called in process_set.cu in branch **AddForcingTerm_CUDA**

Reply:
I have made this work on both branches now using cmake variables